project.md

```diff
@@ -11,41 +11,59 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+I am helping economists (both professionals and students) to be able to perform financial calculations without having to perform them manually, pay for expensive software, use an Google/Excel sheet for just one line, or look for an X formula calculator and use some weird website. With my DSL implementation, their experience could be improved as they could just type one line of code to perform the calculation as they would have in a Google/Excel sheet.
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL is appropriate for the users because it addresses these economists' need to perform calculations without having to open a Google/Excel sheet and have random sheets stored with a few lines of work. Since these users are already having to perform these calculations with some version of typing, whether it be on an online or handheld calculator, they will be able to type a format which follows the conventions of economic formulas and software to input calculations. This means they already have a sufficient technical background with putting variables into a formula so they will not require any more background than is already expected to be able to perform such calculations.
```

Comment:
I agree that for economists and students who are studying econ this could be a great took because they already have the previous knowledge. If you have time towards the end of your project you may want to think about the normal user one that does not have previous background but finds your tool and needs it for some aspect of life. Take me for instance, I hate econ but there may be a time in the future where i could use your tool. Although your dsl does not need this additional layer it may be nice in the future.